codec-http2/src/main/java/io/netty/handler/codec/http2/Http2Stream.java

```diff
@@ -87,16 +116,15 @@
     Http2Stream resetSent();
 
     /**
-     * Indicates whether the remote side of this stream is open (i.e. the state is either
-     * {@link State#OPEN} or {@link State#HALF_CLOSED_LOCAL}).
+     * Indicates whether or not at least one frame has been sent from the local endpoint for this stream.
      */
-    boolean remoteSideOpen();
+    boolean isFrameSent();
```

Comment:
`isFrameSent` is currently implemented as `is headers frame sent`. These two things are not necessarily the same. Should we clarify this method name, or update the implementation in `DefaultHttp2Connection`?